### DISCUSSION TITLE
Soft-deprecated elements

### DISCUSSION THREAD

**Discussion by pan-zhuo** [crystalyragui/MARC2RDA#374] [General]
Are we going to implement the soft-deprecated elements listed in the [Alignment from soft-deprecated to recommended elements](http://www.rdaregistry.info/Aligns/alignSoft2Rec.html)?

In our current mapping, we seem to keep all the 'parallel' elements (parallel title proper, parallel name of publisher, etc.) that are soft-deprecated.

If you look up these elements in the Toolkit, they point to the 'recommended' elements in the Prerecording section.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-01595b77-9820-3a0a-8488-ac6ee64cca09

**Comment by AdamSchiff** [comment]
Interesting... not all of the elements in that Excel spreadsheet are shown as deprecated in the RDA Registry:

[cid:f7820eb3-9eeb-4868-862b-e7c40aceec53]
[cid:c53eafe2-cbf7-45e5-90ca-4dfa4e6d458e]
Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________
From: Zhuo Pan ***@***.***>
Sent: Wednesday, July 27, 2022 5:30 PM
To: uwlib-cams/MARC2RDA ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [uwlib-cams/MARC2RDA] Soft-deprecated elements (Discussion #374)


Are we going to implement the soft-deprecated elements listed in the Alignment from soft-deprecated to recommended elements<http://www.rdaregistry.info/Aligns/alignSoft2Rec.html>?

In our current mapping, we seem to keep all the 'parallel' elements (parallel title proper, parallel name of publisher, etc.) that are soft-deprecated.

If you look up these elements in the Toolkit, they point to the 'recommended' elements in the Prerecording section.
https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-01595b77-9820-3a0a-8488-ac6ee64cca09

**Comment by GordonDunsire** [comment]
My advice is to avoid using any soft-deprecated elements. They are retained to allow RDA communities to move at their own pace in the transition to the official (3R) Toolkit, but the elements are redundant in the context of the Library Reference model. The Registry file is an alignment or cross-walk; it cannot be readily represented by a semantic map.

Of course, such a map might be developed for the context of MARC 21 to RDA, assuming that the legacy of these pre-3R elements is closely reflected in MARC 21 encodings. If so, the transformation package could have an option to use soft-deprecated elements or the recommended elements ... and then the user could use the map to transition from pre-3R elements to recommended elements. However, I would still expect issues with some of the soft-deprecated elements.

Alternatively, some of the alignments are obviously straightforward to map, for example the 'parallel' elements are sub-types of the root element; but it gets complicated after that.

**Comment by briesenberg07** [comment]
*See also sinopia_maps > [Soft-deprecated properties #86](https://github.com/uwlib-cams/sinopia_maps/discussions/86)*
